lib/chutney/TorNet.py

```diff
@@ -764,6 +764,19 @@ def _genRouterKey(self):
             sys.exit(1)
         self._env['fingerprint'] = fingerprint
 
+
+    def _setEd25519Id(self):
+        """Make chutney check for relay microdescriptors before verifying"""
```

Comment:
The function comment describes what the function does, not the feature that it is a part of.

This function is the first step in checking relay microdescriptors, but it only reads the ed25519 key. So a better function comment would be:
```suggestion
        """Read the ed25519 identity key for this router, and set up the 'ed25519-id' entry in the Environ"""
```